### DISCUSSION TITLE
Create sub-path programmatically on package configuration

### DISCUSSION THREAD

**Discussion by manomintis** [adonisjs/core#4416] [Help] [answered]
Is there any conventional way to create a new package.json/tsconfig.json sub-path using package configuration or npm script for version 6? The idea is that some package installed by NPM enables new functionality and a new namespace for importing (like "#something/..."). So it becomes available without forcing a user to create it manually.

**Comment by Julien-R44** [comment, marked as answer]
we don't have any high-level API for that. by high-level API i mean like the one we have with[ Codemods API](https://docs.adonisjs.com/guides/scaffolding#codemods-api) 

that said, you can do it on your own. in your `configure.ts` file you can open and modify files using standard `node:fs`, just as you would with a normal Node script. `readFile` and `writeFile` and you should be able to do what you want

**Reply by manomintis** [reply]
Thanks!
